app/ui/browser/actions/external.js

```diff
@@ -101,7 +101,9 @@ export function menuLocationContext(input, dispatch) {
         pageIndex: -1,
         userTyped: loc,
       }));
-      dispatch(navigatePageTo(-1, fixURL(loc)));
+
+      // TODO this doesn't work anyway, we have to pass in webViewController
+      // dispatch(navigatePageTo(-1, fixURL(loc)));
```

Comment:
I'd be tempted to just delete the feature if it isn't working rather than leave a todo, but go with whatever you think.